scripts/osm2pgsql-replication

```diff
@@ -80,10 +80,10 @@ def connect(args):
     """
     # If dbname looks like a conninfo string use it as such
     if args.database and any(part in args.database for part in ['=', '://']):
-        return psycopg.connect(args.database)
+        return psycopg.connect(args.database, application_name="osm2pgsql-replication")
```

Comment:
I'd rather set `fallback_application_name` so that a user can potentially override with `PGAPPNAME` if they're running multiple DBs

Reply:
Done. 🙂
I didn't know about that funtionality.
Rebased & force pushed.